GifSearcher/GifSearcher/SearchScene/SearchPresenter.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  SearchPresenter.swift
+//  GifSearcher
+//
+//  Created  on 6/4/20.
+//
+
+import UIKit
+
+/**
+ Protocol to pass the search results to the view controller
+ */
+protocol SearchPresentationLogic {
+    func presentSearchResults(response: AnimatedGifSearch.Search.Response)
```

Comment:
I think a `SearchResponse` protocol here could decouple from `AnimatedGifSearch`.

Such a protocol might need an `associatedtype` to define the type of data returned, which would complicate things, though.